Content.Server/Body/Components/BodyComponent.cs

```diff
@@ -83,9 +83,10 @@ protected override void Startup()
             }
         }
 
-        public override void Gib(bool gibParts = false)
+        public override List<SharedBodyPartComponent> Gib(bool gibParts = false)
```

Comment:
this is a perennial issue with body stuff (storing comps instead of entities) but these should return `HashSet<EntityUid>` rather than `List<SharedBodyPartComponent>`